src/components/Banner/Banner.scss

```diff
@@ -1,133 +0,0 @@
-@import "../../styles/themes";
-@import "../../styles/typography";
-@import "../../styles/global-css-settings";
-
-
-.banner {
```

Comment:
no neet to convert banner to css modules, we going to remove this component very soon